app/models/variant.rb

```diff
@@ -67,6 +67,15 @@ def display_name
     name
   end
 
+  def allele_registry_id
+    response = AlleleRegistry.new(id).response
+    if response.instance_of?(String)
+      JSON.parse(response)['@id'].split('/')[-1]
```

Comment:
This will die if we don't get a response or `@id` isn't present. To preserve the current behavior in case of this sort of failure you could do something like `JSON.parse(response)['@id'].split('/')[-1] rescue nil` which will just catch all a exceptions and return nil if one is thrown.

Reply:
I'll add that real quick.